setup.cfg

```diff
@@ -58,6 +58,9 @@ ignore_missing_imports = True
 [mypy-connexion]
 ignore_missing_imports = True
 
+[mypy-skimage]
```

Comment:
@dishmop please remove this here. Only `aea` package dependencies should be listed here. To ignore the `skim age` in the code static checks add `type: ignore` behind the import.